devops/jobs/RunRemoteAnsiblePlaybook.groovy

```diff
@@ -0,0 +1,105 @@
+/*
+
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    SSH_AGENT_KEY: Key for the SSH Agent
+    CONFIGURATION_REPO: standard default available
+    CONFIGURATION_BRANCH: defaults to master
+    SSH_USER: ansible connect user - defaults to ubuntu.  Make sure this matches up with SSH_AGENT_KEY
+    ANSIBLE_PLAYBOOK:  The playbook used in this job
+
+    Expected credentials - these will normally be set up on the Folder.
+        jnkins-aws-credentials: a file credential that can be set in the environment as AWS_CONFIG_FILE for assuming role
+        find-host-role-arn: ARN of the IAM role that will be assumed
+    Optional:
+    ACCESS_CONTROL: list of Jenkins matrix targets who should get read access (edx*team or username)
+    SCHEDULE: Cron syntax consumed by Jenkins triggered builds
+    NOTIFY_ON_FAILURE: email address to be notified when this job fails
+*/
+package devops.jobs
+
+import static org.edx.jenkins.dsl.DevopsConstants.common_wrappers
+import static org.edx.jenkins.dsl.DevopsConstants.common_logrotator
+import static org.edx.jenkins.dsl.DevopsConstants.common_read_permissions
+
+class RunRemoteAnsiblePlaybook {
+    public static def job = { dslFactory, jobName, extraVars ->
+        return dslFactory.job("${jobName}") {
+
+            // Common Jenkins Job features
+            wrappers common_wrappers
+
+            wrappers {
+              sshAgent(extraVars.get("SSH_AGENT_KEY", ''))
+            }
+
+            parameters {
+                stringParam('CONFIGURATION_REPO', extraVars.get('CONFIGURATION_REPO', 'https://github.com/edx/configuration.git'),
+                            'Git repo containing the analytics pipeline configuration automation.')
+                stringParam('CONFIGURATION_BRANCH', extraVars.get('CONFIGURATION_BRANCH', 'master'),
+                            'e.g. tagname or origin/branchname')
+                stringParam("ANSIBLE_INVENTORY",
+                            "", "The inventory address to run the remote script")
```

Comment:
Maybe reword as "The inventory address(es) where we run the remote script"  (You can provide multiple hosts to an inventory, hence my weird (es) phrasing).